### PR TITLE
Horizontal scroll

### LINKED ISSUE #177: Horizontal scrolling using Shift+scroll
### Motivation

Spreadsheets have largely been the most common app users need to navigate horizontally. Unfortunately Quadratic seems to only offer horizontal scrolling by arrow keys navigation. So quick mouse navigation is not possible (there are no scrollbars which is IMHO a good UX decision but there must be a "mouse way" to scroll horizontally without putting your hand elsewhere from the mouse).

### Envisioned solution

Make `Shift+scroll` work as horizontal scroll :wink:.

### PR DESCRIPTION
**Pull request by juanbercoff**
- fix #177 
Extracted keys to array
Added horizontal scroll keys
If both keys  types are pressed no action is taken

### CODE REVIEW

**Comment by juanbercoff** [comment]
> Figma seems to disable this behavior on MacOS, but it exists on Windows and Linux.
> 
> This is probably what we should do as well.

Done in last commit

File: src/gridGL/pixiOverride/Wheel.ts [hunk 1]
```diff
@@ -293,13 +313,14 @@ export class Wheel extends Plugin {
     } else if (e.ctrlKey && this.options.trackpadPinch) {
       this.pinch(e);
     } else {
+      const isMac = navigator.userAgent.search('Mac') !== -1;
```
Comment (davidkircos) [inline src/gridGL/pixiOverride/Wheel.ts:316]: We have a function for this in `quadratic/src/utils/isMac.ts`, could you switch to using that function @juanbercoff

**Review by davidkircos — CHANGES_REQUESTED** [review]

**Comment by HactarCE** [comment]
Working well on Windows!

File: src/gridGL/pixiOverride/Wheel.ts (hunk 1 above)
Comment (juanbercoff) [reply src/gridGL/pixiOverride/Wheel.ts]: Done!

**Pull request merged**